src/cromshell/abort/command.py

```diff
@@ -46,8 +46,5 @@ def main(config, workflow_ids):
                 # command to abort remaining workflows.
                 raise_exception=False,
             )
-        else:
-            # Todo: Replace input with requests_out.json() once rebased with submit PR
-            io_utils.pretty_print_json(requests_out.text)
 
-    return 0 if return_code == 0 else 1
+    return return_code
```

Comment:
Giving this a 2nd thought, I'd vote for a small patch to this PR (so you can deal with it here, or in a separate small PR): register the requests that were `not ok`, and tell the users all together towards the end.
Here the failures are communicated to the user on-the-spot, which can be hard to keep track of.
Anyway, consider this a feature request.